### PR TITLE
Delivery report coverage and completeness (täckningsgrad) from panel genes

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
End user representative: ES

Change the täckningsgrad also to show from gene panel before updating the coverage showing from gene panel.

- Test on stage (scout, cg)
- Merge
- Deploy

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
How to test: 
1. go to rasta
2. hiseq-user 
3. us
4. cg upload scout --print [familyid] 
5. check the coverage numbers in the output

**Comment by patrikgrenfeldt** [comment]
@ingkebil could you review the code on this one?

File: tests/meta/report/conftest.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def get_post_analysis_files(self, family: str, version, tags: [str]):
                    '_config.yaml'
         elif tags[0] == 'sampleinfo':
             path = '/mnt/hds/proj/bioinfo/bundles/' + family + '/2018-01-30/' + family + \
-                   '_qc_sample_info.yaml'
+                    '_qc_sample_info.yaml'
```
Comment (ingkebil) [inline tests/meta/report/conftest.py]: pep8 mishap? :)

File: cg/meta/report/api.py [hunk 2]
```diff
@@ -282,3 +289,13 @@ def _incorporate_lims_data(self, report_data: dict):
             sample['application'] = ReportAPI._present_string(lims_sample.get('application'))
             sample['application_version'] = lims_sample.get('application_version')
             sample['received'] = ReportAPI._present_date(lims_sample.get('received'))
+
+    def _get_genes_from_scout(self, panels) -> list:
```
Comment (ingkebil) [inline cg/meta/report/api.py]: We seem to be using type hints consistently throughout the code. You can add a typehint for `panels` as well

File: tests/meta/report/conftest.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def get_post_analysis_files_root_dir(self):
 class MockChanjo:
     _sample_coverage_returns_none = False
 
-    def sample_coverage(self, sample_id) -> dict:
+    def sample_coverage(self, sample_id, genes) -> dict:
```
Comment (ingkebil) [inline tests/meta/report/conftest.py]: Missing type hints

File: tests/meta/report/conftest.py [hunk 4]
```diff
@@ -200,21 +200,31 @@ def application(self, tag: str):
...
         self.lims = lims_api
         self.deliver = deliver_api
         self.chanjo = chanjo_api
         self.analysis = analysis_api
+        self.scout = scout_api
         self.LOG = logger
         self.yaml_loader = yaml_loader
         self.path_tool = path_tool
 
+    def _open_bundle_file(self, file_path):
```
Comment (ingkebil) [inline tests/meta/report/conftest.py]: is this used somewhere?

**Comment by patrikgrenfeldt** [comment]
@emiliaol please test again, the database was broken due to me and Maya developing on it yesterday, sorry about that

**Comment by patrikgrenfeldt** [comment]
Now it says: "Resultat baserade på valda genpaneler - $GENE PANELS.".
A bit unsure if I like the dot or not :)

**Comment by patrikgrenfeldt** [comment]
Have you seen it working with more than one selected gene panel?

**Comment by patrikgrenfeldt** [comment]
@ingkebil I think I've done what you requested, could you approve the PR now?

**Pull request merged**